lib/Macros/Sources/SwiftMacros/SwiftifyImportMacro.swift

```diff
@@ -1316,6 +1333,56 @@ func isMutableSpan(_ type: TypeSyntax) -> Bool {
   return name == "MutableSpan" || name == "MutableRawSpan"
 }
 
+func isImmutableSpan(_ type: TypeSyntax) -> Bool {
+  if let optType = type.as(OptionalTypeSyntax.self) {
+    return isImmutableSpan(optType.wrappedType)
+  }
+  if let impOptType = type.as(ImplicitlyUnwrappedOptionalTypeSyntax.self) {
+    return isImmutableSpan(impOptType.wrappedType)
+  }
+  if let attrType = type.as(AttributedTypeSyntax.self) {
+    return isImmutableSpan(attrType.baseType)
+  }
+  guard let identifierType = type.as(IdentifierTypeSyntax.self) else {
+    return false
+  }
+  let name = identifierType.name.text
+  return name == "Span" || name == "RawSpan"
+}
+
+func getAvailability(_ newSignature: FunctionSignatureSyntax, _ typeAvailability: [String: String]?)
+  throws -> [AttributeListSyntax.Element] {
+  guard let typeAvailability else {
+    return []
+  }
+  var hasMutableSpan = false
+  var hasSpan = false
+  if let returnClause = newSignature.returnClause {
+    hasMutableSpan = isMutableSpan(returnClause.type)
+    hasSpan = isImmutableSpan(returnClause.type)
+  }
+  for param in newSignature.parameterClause.parameters {
+    if hasMutableSpan {
+      break
+    }
+    hasMutableSpan = isMutableSpan(param.type)
+    hasSpan = hasSpan || isImmutableSpan(param.type)
+  }
+  if !hasMutableSpan && !hasSpan {
+    return []
+  }
+  let availability =
+    if hasMutableSpan {
+      typeAvailability["MutableSpan"]
+    } else {
+      typeAvailability["Span"]
```

Comment:
These were both implemented in the standard library at the same time, so you shouldn't need to handle them separately.